File: qa/qa-js/tests/test_login_web.js
const { chromium } = require('playwright');

(async () => {
    const browser = await chromium.launch({ headless: false });
    const page = await browser.newPage();

    await page.goto("https://the-internet.herokuapp.com/login");

    await page.fill("#username", "tomsmith");
    await page.fill("#password", "SuperSecretPassword!");
    await page.click('button[type="submit"]');

    const mensagem = await page.textContent("#flash");

    if (mensagem.includes("You logged into a secure area!")) {
        console.log("✔ LOGIN funcionando!");
    } else {
        console.log("❌ LOGIN falhou.");
    }

    await browser.close();
})();

File: qa/qa-python/test_login_web.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.chrome.options import Options

def test_login_web():
    options = Options()
    options.add_argument("--headless")
    driver = webdriver.Chrome(options=options)

    driver.get("https://the-internet.herokuapp.com/login")

    driver.find_element(By.ID, "username").send_keys("tomsmith")
    driver.find_element(By.ID, "password").send_keys("SuperSecretPassword!")
    driver.find_element(By.TAG_NAME, "button").click()

    assert "You logged into a secure area!" in driver.page_source

    driver.quit()
